webapi/src/main/scala/org/knora/webapi/config/AppConfig.scala

```diff
@@ -27,25 +34,64 @@ final case class AppConfig(
   jwtLongevity: String,
   cookieDomain: String,
   allowReloadOverHttp: Boolean,
-  knoraApi: KnoraAPI,
+  fallbackLanguage: String,
+  knoraApi: KnoraApi,
```

Comment:
Not changing this to DSP?

Reply:
I didn't want to touch the naming of configurations in this PR. Instead, I made a new issue for that, so we can handle it globally: https://dasch.atlassian.net/browse/DEV-1326